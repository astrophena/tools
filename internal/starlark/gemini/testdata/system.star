# Â© 2024 Ilya Mateyko. All rights reserved.
# Use of this source code is governed by the ISC
# license that can be found in the LICENSE.md file.

responses = gemini.generate_content(
    model="gemini-1.5-flash",
    contents=[("user", "Write a story about very kind heiress and her maid.")],
    system_instructions="Write in Russian.",
)

print(responses[0][0])
