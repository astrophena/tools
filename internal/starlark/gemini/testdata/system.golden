Агата, наследница огромного состояния, была удивительно доброй.  Её богатство измерялось не только золотом и бриллиантами, но и бесконечным потоком доброты, который она щедро раздавала окружающим.  Её дом, роскошный особняк на берегу моря, всегда был полон гостей – не только знатных особ, но и нуждающихся, которых Агата принимала с одинаковой теплотой.  Её единственной слугой была Мария, молодая женщина с грустными глазами и тихим голосом.  Мария работала у Агаты не за деньги – зарплата была щедрой, но это не было главной причиной её преданности.

Мария приехала в особняк, сломленная горем.  Смерть родителей оставила её без средств к существованию, и она, почти отчаявшись, обратилась за помощью к Агате, о которой слышала как о женщине с добрым сердцем. Агата приняла её не как прислугу, а как подругу, разделив с ней своё горе и предложив не только работу, но и кров.

В доме Агаты не было строгой иерархии. Агата помогала Марии по хозяйству, смеясь над своими неуклюжими попытками натереть серебро. Они вместе готовили варенье из ягод, собранных в саду, и вечерами читали книги у камина, делясь друг с другом переживаниями.  Агата рассказывала Марии о своих благотворительных проектах, а Мария делилась своими мечтами о тихом семейном счастье, о котором она всегда мечтала.

Однажды, богатый и влиятельный герцог предложил Агате руку и сердце.  Агата, привыкшая к свободе и независимости, колебалась.  Она ценила его богатство и положение, но не чувствовала к нему настоящей любви.  Мария, видя её сомнения, мягко сказала: «Агата, ваше сердце должно выбирать, а не ваш ум».  И Агата, послушавшись своего сердца и совета Марии, отказала герцогу.

Вместо этого, Агата  пожертвовала значительную часть своего состояния на создание приюта для сирот,  называя его в честь Марии.  Мария, тихо радуясь, стала его заведующей, воплощая в жизнь свою мечту о заботе о нуждающихся.  Агата же,  оставшись в своём особняке, не была одинока.  Её жизнь наполнялась  смыслом,  и  теплота её сердца, разделяемая с Марией,  сияла ярче любых бриллиантов.  Их дружба,  зародившаяся в горе,  превратилась в  нерушимую связь,  доказав, что настоящая ценность не в богатстве, а в доброте и верности.
